compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/EnvironmentResolver.java

```diff
@@ -114,9 +115,14 @@ public SymbolEnv lookUp(BLangCompilationUnit compilationUnit, LinePosition lineP
 
     @Override
     public void visit(BLangCompilationUnit compUnit) {
-        compUnit.getTopLevelNodes().stream().filter(
-                node -> !(node instanceof BLangFunction && ((BLangFunction) node).flagSet.contains(Flag.WORKER)))
-                .forEach(topLevelNode -> this.acceptNode((BLangNode) topLevelNode, this.symbolEnv));
+
```

Comment:
Are these new lines getting added by the IDE? If so, please modify the formatting settings to avoid adding new lines at the start of a method.

Reply:
Fixed.